src/python/build_management/build_manager.py

```diff
@@ -864,6 +864,38 @@ def setup(self):
     return True
 
 
+class CuttlefishKernelBuild(RegularBuild):
+  """Represents a Android Cuttlefish kernel build."""
+
+  def setup(self):
+    """Android kernel build setup."""
+    from platforms.android import adb
+
+    result = super().setup()
+    if not result:
+      return result
+
+    environment.set_value('VMLINUX_PATH', self.build_dir)
+
+    cvd_dir = environment.get_value('CVD_DIR')
+    adb.stop_cuttlefish_device()
+
+    for image_filename in ['bzImage', 'initramfs.img']:
```

Comment:
nit: put `['bzImage', 'initramfs.img']` as a class constant (and use a tuple instead).

i.e.

```python
class CuttlefishKernelBuild(..):

  _IMAGE_FILES = ('bzImage', 'initramfs.img')
```